rpc/src/methods.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+//! Module contains RPC method handlers for Full Node JSON-RPC interface
+use crate::views::AccountView;
+use anyhow::{ensure, Result};
+use core::future::Future;
+use libra_types::{account_address::AccountAddress, account_config::AccountResource};
+use serde_json::Value;
+use std::collections::HashMap;
```

Comment:
can we combine std imports? like `std::{collections::HashMap, convert::TryFrom, ... }`